# Max torque is 25 N-m
float64 torque_max
# Torque angle should be between 0 and pi/2
float64 torque_angle
# Adjust values should be between 0 and 1, both included
float64 roll_adjust
float64 pitch_adjust